deku-derive/src/macros/deku_read.rs

```diff
@@ -77,7 +77,7 @@ fn emit_struct(input: &DekuData) -> Result<TokenStream, syn::Error> {
                 let __deku_pad = 8 * ((__deku_rest.len() + 7) / 8) - __deku_rest.len();
                 let __deku_read_idx = __deku_input_bits.len() - (__deku_rest.len() + __deku_pad);
 
-                Ok(((__deku_input_bits[__deku_read_idx..].as_raw_slice(), __deku_pad), __deku_value))
+                Ok(((__deku_input_bits[__deku_read_idx..].domain().region().unwrap().1, __deku_pad), __deku_value))
```

Comment:
I'd much rather prefer naming the tuple field.

```suggestion
                let (_, remaining_bytes, _) = __deku_input_bits[__deku_read_idx..].domain().region().unwrap();
                Ok(((remaining_bytes, __deku_pad), __deku_value))
```

Reply:
I could also just make the tuples an actual named struct here.